checkov/terraform/checks/resource/aws/MQBrokerNotPubliclyExposed.py

```diff
@@ -0,0 +1,24 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class MQBrokerNotPubliclyExposed(BaseResourceCheck):
```

Comment:
@mattfenwick I seems this class can inherit from `BaseResourceNegativeValueCheck` - just specify the path to the bad key and it will do the rest

Reply:
👍